CHANGELOG.md

```diff
@@ -1,37 +1,36 @@
 # Changelog
 
-## [Unreleased]
+## [1.113.0] - 2023-04-18
 
-### Changes
-- Increase MSRV to 1.65.0. #4236
-- Remove upper limit on the attachment size. #4253
-- Update rPGP to 0.10.1. #4236
-- Compress `mime_headers` column with HTML emails stored in database
-- Strip BIDI characters in system messages, files, group names and contact names #3479
-- maybe_add_time_based_warnings(): Use release date instead of the provider DB update one
-- Remove confusing log line "ignoring unsolicited response Recent(…)" #3934
-- Cleanly terminate deltachat-rpc-server.
-  I.e. on EOF in stdin.
-- Refactorings #4317
-- Add JSON-RPC API `can_send()`.
+### Added
+- New JSON-RPC API `can_send()`.
 - New `dc_get_next_msgs()` and `dc_wait_next_msgs()` C APIs.
   New `get_next_msgs()` and `wait_next_msgs()` JSON-RPC API.
   These APIs can be used by bots to get all unprocessed messages
   in the order of their arrival and wait for them without relying on events.
-- Async Python API `get_fresh_messages_in_arrival_order()` is deprecated
-  in favor of `get_next_msgs()` and `wait_next_msgs()`.
 - New Python bindings API `Account.wait_next_incoming_message()`.
 - New Python bindings APIs `Message.is_from_self()` and `Message.is_from_device()`.
-- Remove metadata from avatars and JPEG images before sending #4037
-- Reduce + recode images to JPEG if they are > 500K in size #4037
-- Gracefully terminate deltachat-rpc-server on ctrl-c and SIGTERM.
+
+### Changes
+- Increase MSRV to 1.65.0. #4236
+- Remove upper limit on the attachment size. #4253
+- Update rPGP to 0.10.1. #4236
+- Compress HTML emails stored in the `mime_headers` column of the database.
+- Strip BIDI characters in system messages, files, group names and contact names. #3479
+- Use release date instead of the provider database update date in `maybe_add_time_based_warnings()`.
+- Gracefully terminate `deltachat-rpc-server` on Ctrl+C (`SIGINT`), `SIGTERM` and EOF.
+- Async Python API `get_fresh_messages_in_arrival_order()` is deprecated
+  in favor of `get_next_msgs()` and `wait_next_msgs()`.
+- Remove metadata from avatars and JPEG images before sending. #4037
+- Reduce + recode images to JPEG if they are > 500K in size. #4037
```

Comment:
```suggestion
- Recode PNG and other supported image formats to JPEG if they are > 500K in size. #4037
```

Reply:
JPEGs and other images are also reduced in linear size if they are > 500K. That's why the word "Reduce" was here

Reply:
i see :) tbh, i found the original entry very confusing and hard to understand, but maybe this is only me. also, i think, the changelog should point out changes mainly - and scaling down JPEG was always the case.

but yes, we can improve, maybe say ` Recode PNG and other supported image formats to JPEG if they are > 500K in size (JPEG images will be scaled down as before) #4037` or so - feel free to add a suggestion.

Reply:
Yes, scaling JPEGs down was before but only if a JPEG exceeds the limit on linear size, not the byte size. I think we can add this note on the next line, it's probably less important than changing an image format